src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,22 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-radius * 2;
+  height: $stopwatch-radius * 2;
+  border: 1px solid grey;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    @include watch-element(1, 8px, 40%, red, 50% 50% 0 0);
+    animation: rotate-hand 3600s steps(60, end) infinite;
+  }
+
+  &__seconds-hand {
+    @include watch-element(2, 8px, 47%, blue, 50% 50% 0 0);
```

Comment:
it's not clear what is 40% and 47%, can you make the corresponding variables for such values to be able to see the calculations?

Reply:
Fixed.